--- kguitar-0.5/doc/en/index.docbook.old	2005-06-25 20:11:38.000000000 +0200
+++ kguitar-0.5/doc/en/index.docbook	2005-06-25 20:11:57.000000000 +0200
@@ -2,6 +2,7 @@
 <!DOCTYPE book PUBLIC "-//KDE//DTD DocBook XML V4.1-Based Variant V1.0//EN" "dtd/kdex.dtd" [
   <!ENTITY kguitar '<application>KGuitar</application>'>
   <!ENTITY kapp "&kguitar;">
+  <!ENTITY package "kguitar">
   <!ENTITY % addindex "IGNORE">
   <!ENTITY % English "INCLUDE"><!-- change language only here -->
   
