--- kguitar-0.4.9/kguitar/Makefile.am.orig	Thu Aug 28 04:05:37 2003
+++ kguitar-0.4.9/kguitar/Makefile.am	Sat Jan  3 22:16:01 2004
@@ -103,11 +103,13 @@
 
 
 $(INSTALL_KGTABS):
-	@echo "Copy kgtabs.tex to $(TEXMF)/tex/generic/kgtabs"
-	$(mkinstalldirs) $(TEXMF)/tex/generic/kgtabs
-	$(INSTALL_DATA) $(srcdir)/kgtabs.tex $(TEXMF)/tex/generic/kgtabs/kgtabs.tex
-	@echo "Running texhash..."
-	@texhash
+	@echo "Copy kgtabs.tex to $(DESTDIR)$(TEXMF)/tex/generic/kgtabs"
+	$(mkinstalldirs) $(DESTDIR)$(TEXMF)/tex/generic/kgtabs
+	$(INSTALL_DATA) $(srcdir)/kgtabs.tex $(DESTDIR)$(TEXMF)/tex/generic/kgtabs/kgtabs.tex
+	@if test "$(DESTDIR)" = ""; then \
+	echo "Running texhash..."; \
+	texhash ; \
+	fi
 
 $(UNINSTALL_KGTABS):
 	@echo "Delete kgtabs.tex"
